packages/contracts/truffle-config.js

```diff
@@ -3,20 +3,16 @@ module.exports = {
   // to customize your Truffle configuration!
   migrations_directory: "build/migrations",
   test_directory: "build/test",
-  mocha: {
-    reporter: 'mocha-multi-reporters',
-    reporterOptions: {
-      reporterEnabled: "mocha-junit-reporter, spec",
-      mochaJunitReporterReporterOptions: {
-        mochaFile: './build/junit/junit.xml'
-      }
-    }
-  },
   networks: {
     develop: {
       host: "localhost",
-      port: 9545,
+      port: 8545,
```

Comment:
I specifically had to make this 9545 for it to work with `truffle develop`, it's working at 8545?

Reply:
Just checked, seems so